frontend/packages/model-registry/upstream/api/openapi/mod-arch.yaml

```
@@ -149,7 +149,7 @@ paths:
       responses:
         "200":
           # Define a response that might include credentials
-          $ref: "#/components/responses/ModelRegistryKindResponse" 
+          $ref: "#/components/responses/ModelRegistryAndCredentialsResponse" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)